paasta_tools/cli/cmds/status.py

```diff
@@ -1124,6 +1132,112 @@ def print_flink_status(
     return 0
 
 
+def print_kubernetes_status_v2(
+    cluster: str,
+    service: str,
+    instance: str,
+    output: List[str],
+    status,
```

Comment:
should we annotate `status` here, `replicasets` at https://github.com/Yelp/paasta/pull/3052/files?file-filters%5B%5D=.py#diff-36175af498b294330c337132851420b6d63c7469e72162fb6a4496fcd5cbaff2R1187 (and further below) and `pods` at https://github.com/Yelp/paasta/pull/3052/files?file-filters%5B%5D=.py#diff-36175af498b294330c337132851420b6d63c7469e72162fb6a4496fcd5cbaff2R1237? 

https://mypy.readthedocs.io/en/latest/common_issues.html?highlight=reveal_type#displaying-the-type-of-an-expression will probably help get the types of these

Reply:
I would assume that `replicasets` is `V1ReplicaSet` and `pods` is `V1Pod` tho

Reply:
This is the data being returned by the Paasta API, so they're actually OpenAPI generated models, but I can look into annotating -- shouldn't be too hard